src/debug_toolkit/payloads/stack_trace.py

```diff
@@ -1,19 +1,45 @@
 def entrypoint(output_path: str):
-    import threading
-    import sys
+    import time
+    import json
 
     ALL_THREADS = bool(ALL_THREADS_PLACEHOLDER)
-    stack_trace = get_traceback(ALL_THREADS)
-    for thr in threading.enumerate():
-        if sys.version_info[0] >= 3 and sys.version_info[1] >= 8:
-            #  _native_id is only from python3.8+
-            replacement_str = f"{thr.getName()} tid:{thr._native_id}"
-        else:
-            replacement_str = f"{thr.getName()} thread identity:{thr._ident}"
-        thread_id_str = '0x{:016x}'.format(thr._ident)
-        stack_trace = stack_trace.replace(thread_id_str, replacement_str)
+    AMOUNT = int(AMOUNT_PLACEHOLDER)
+    SLEEP_DURATION_S = int(SLEEP_DURATION_S_PLACEHOLDER)
+
+    thread_stack_dumps=[]
+    for _ in range(AMOUNT):
```

Comment:
I think the code in the loop may throw an exception.
Should we wrap it in try/except ?